rmgpy/rmg/main.py

```diff
@@ -1473,9 +1491,10 @@ def initializeReactionThresholdAndReactFlags(self):
         numCoreSpecies = len(self.reactionModel.core.species)
 
         # Number of families in RMG
+        rmg_families = list(self.database.kinetics.families.keys())
         num_families = len(self.database.kinetics.families)
 
-        if self.filterReactions:
+        if self.filterReactions or self.restart_seed_path:
```

Comment:
I think that would mean that previously unreacted things would never be reacted in the restarted simulation, since the flags would never be updated.

Not exactly sure that the optimal behavior is though.

Reply:
Implemented